doc/source/migrations/index.rst

```diff
@@ -12,17 +12,18 @@ Development version
 
 **New features:**
 
-- Obscuring vulnerabilities results in ``ansys/action/check-vulnerabilities``. This is useful when you want to hide the
+- Obscuring vulnerabilities results in ``ansys/actions/check-vulnerabilities``. This is useful when you want to hide the
```

Comment:
Here and in the next bullet, a noun should follow the content in double backticks to indicate what that code entity is. Something like ""Obscuring vulnerabilities results in the ``ansys/actions/check-vulnerabilities`` action running." (Although I'm not sure that this is exactly what you are trying to say.)